.github/workflows/test.yml

```diff
@@ -1,5 +1,11 @@
+---
 name: Tests
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+    branches:
+      - master
```

Comment:
should we also include develop?

Reply:
Not necessary, because we run tests on `push`, which covers everything internal to this repo. The pull request tests are only relevant if we get a pull request from a repo that doesn't have the same push tests - which will normally target `master` anyway.